inc/modules/import/epub/class-epub201.php

```diff
@@ -554,4 +580,41 @@ protected function kneadHref( \DOMDocument $doc, $type, $href ) {
 
 		return $doc;
 	}
+
+
+		/**
+		 * Parse manifest with type 'application/xhtml+xml' to array
+		 *
+		 * @param \SimpleXMLElement $xml
+		 * @return array()
```

Comment:
If you are modifying `$this->manifest` then there is no need to return it.

Reply:
Removed the return value. We had done it considering method chaining in future implementations if needed.